lib/auth/bot.go

```diff
@@ -281,6 +282,7 @@ func (s *Server) checkOrCreateBotToken(ctx context.Context, req *proto.CreateBot
 					types.JoinMethodIAM,
 					types.JoinMethodGitHub,
 					types.JoinMethodCircleCI,
+					types.JoinMethodKubernetes,
```

Comment:
since tbot support has been moved out of this PR I think changes in this file should be reverted so that we don't suggest it as a supported method here